### ISSUE TITLE
Look into creating renewable snapshot without developers

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/nrrd#1921]
To avoid taking developers time from onrr.gov, we can build the approved prototype using word or InDesign for this year and look into automating it next year. Also this will only need to be done once a year so it's not worth taking time away from goals & priorities.

**Comment by Maroyafaied** [comment]
@cthomasONRR @mcharg try word. @lpgoldstein @jkwonatx try InDesign.

**Comment by Maroyafaied** [comment]
@lpgoldstein @cthomasONRR I agree it looks great, the table heading error would need to be fixed. The table summaries doesn't need to be if everything else passes but it would be best if we can get that resolved as well.

**Comment by mcharg** [comment]
Regina also contradicted Sid about table summary and said it is actually needed.

**Comment by mcharg** [comment]
@lpgoldstein Do you have any control over the margins in the pdf?  When I print it out it adds big margins on the right and bottom.  See the attached print to PDF that shows what it looks like.
[Snapshot Printed to PDF](https://github.com/ONRR/nrrd/files/8017894/Renewables.Snapshot.FY20.pdf)

**Comment by lpgoldstein** [comment]
@Maroyafaied @mcharg I will try to fix those in Excel. @mcharg I'll work on the margins as well. Thanks for printing  it out.

**Comment by cthomasONRR** [comment]
I tried to resolve the Table Summary error like I normally do, and it doesn't work on this doc because the 'tables' aren't really tables but file attachments. I think Adobe is struggling to understand if they are indeed tables, or need to be reclassified as something else.

**Comment by lpgoldstein** [comment]
@cthomasONRR Exactly

**Comment by cthomasONRR** [comment]
Maybe this should not be a PDF. Could we present it as a locked Powerpoint?

**Comment by lpgoldstein** [comment]
That would be the easiest for us and Shannon and I were talking about that on Friday. Let me mess with the excel files and see if that helps. @Maroyafaied opinions on a locked powerpoint?

**Comment by Maroyafaied** [comment]
I'll look into fixing the accessibility issue and let you know. I was able to get it to pass for table heading error so I'll work on the table summary now.

**Comment by Maroyafaied** [comment]
Here's the pdf with everything passing except the table summary.
[Renewables Snapshot FY2020.pdf](https://github.com/ONRR/nrrd/files/8018265/Renewables.Snapshot.FY2020.pdf)

**Comment by Maroyafaied** [comment]
@cthomasONRR Ya sorry, the attachment above passes on everything on my end. @mcharg or @lpgoldstein can you also run an accessibility check and make sure
[Renewables.Snapshot.FY2020 acc.pdf](https://github.com/ONRR/nrrd/files/8018326/Renewables.Snapshot.FY2020.acc.pdf)
?

**Comment by lpgoldstein** [comment]
@Maroyafaied It says summary table skipped for me.

**Comment by Maroyafaied** [comment]
@lpgoldstein you have to make sure it's checked when you run the accessibility check so it checks it and not skip it  
![image](https://user-images.githubusercontent.com/40175688/152863425-658ae141-0117-4378-aca7-6e165f03206f.png)

**Comment by cthomasONRR** [comment]
So weird, it passes all checks for me:
![image](https://user-images.githubusercontent.com/72526208/152863620-08756179-e8ba-45ad-af03-c60aba660303.png)

**Comment by lpgoldstein** [comment]
passed and trying to print to see how the margins work for me

**Comment by lpgoldstein** [comment]
![IMG_9964](https://user-images.githubusercontent.com/39419003/152870251-d2b0d348-788e-4461-9c75-7652bfb2f294.jpg)
@mcharg mine printed ok without changing anything on the margins

**Comment by Maroyafaied** [comment]
There's this https://www.doi.gov/priorities/clean-energy-future

**Comment by cthomasONRR** [comment]
@Maroyafaied I was researching too and just found the same link! That one has BOEM and BLM links at the bottom of the page, I think it might be a good candidate. I'll keep looking too.

**Comment by cthomasONRR** [comment]
Here a link to the full DOI strategic plan, but I prefer the link Maroya found. This PDF might be too much info and renewables is only the first mission area discussed. https://www.doi.gov/sites/doi.gov/files/uploads/fy2018-2022-strategic-plan.pdf
